### DISCUSSION TITLE
A place for community links?

### DISCUSSION THREAD

**Discussion by edthedev** [uillinois-community/uillinois-community.github.io#24] [General] [answered]
We have some GitHub resources around campus that are maintained by campus users. 

in example: 
https://dlstrong.github.io/git-novice/

It could be interesting to use this space to allow community members to share these, but I'm unsure how to proceed. 
Normally I would toss a page into the 'Wiki' section here, but I see we discussed avoiding that approach for this space.

Thoughts on where to put such resources and whether they belong here?

**Comment by areynold** [comment, marked as answer]
I think it would be reasonable to add them to this repo, much like Dena is doing with the link above. Just put in a pull request adding a new section or page. I think the key will be contextualizing the links.

For example, the hands-on training offered by the Software Carpentry group is what makes this link unique and relevant. The course material duplicates a lot of github's own documentation, so I would rather point upstream if we're just answering "how do I do x".

All in my opinion, of course. Thoughts?

**Comment by edthedev** [comment]
We've landed on gathering these on this page.
https://github.com/uillinois-community/uillinois-community.github.io/blob/main/resources/index.md
